encoding/scenefiltering.md

```diff
@@ -60,16 +60,17 @@ time. At this point, your script should look something like this:
 import vapoursynth as vs
 core = vs.get_core()
 
-src = core.lsmas.LWLibavSource("source.mkv")
-filtered = default_filtering(src)
+src = core.lsmas.LWLibavSource("source.m2ts")
+filtered = core.resize.Bilinear(src, width=1280, height=720)
 
-light_denoise = someDenoiseFilter(filtered)
-heavy_denoise = someOtherDenoiseFilter(filtered)
+light_denoise = some_denoise_filter(filtered)
+heavy_denoise = some_other_denoise_filter(filtered)
 
 aa = antialiasing(filtered)
 
-light_deband = deband1(filtered)
-med_deband   = deband2(filtered)
+default_deband = deband(filtered)
+light_deband   = deband1(filtered)
+medium_deband  = deband2(filtered)
```

Comment:
This is more of a principal thing, but wouldn't you deband the denoised clip? In which case the pipeline becomes more like this:

1. denoise
1. (scenefilter denoising)
1. deband
1. (scenefilter debanding)
1. (re-add noise/grain)

Actually, the next example does it like I propose here, so it's just this example here that needs work.

Reply:
Do you mean the fact that I am applying the deband to `filtered` here? Because I was under the impression that denoise, aa, deband is the correct order if you need to aa the clip.

If it's the `filtered` thing, that's because this example is just for creating the base filters. But I should probably change these so they look more similar to the next block.

Reply:
My main point is that using `filtered` here invalidates the pipeline because when you rfs later you only frames that are either debanded or denoised but not both.